packages/techdocs-common/__mocks__/aws-sdk.ts

```diff
@@ -71,11 +71,7 @@ export class S3 {
     return {
       promise: () =>
         new Promise((resolve, reject) => {
-          if (!fs.existsSync(Key)) {
```

Comment:
isn't this probably still good to have

Reply:
It is and should! But currently fails on Windows - since TechDocs publishers are not well written to work on Windows. (Wrote more about it in PR description).